### PR TITLE
feat: Add Material-UI Cozy Theme 🎉

### PR DESCRIPTION
**Pull request by kosssi**
You can now use Material-UI directly on your cozy app with this cozy theme. Only React app can use it.

An exemple:
```jsx
import MuiCozyTheme from 'cozy-ui/react/MuiCozyTheme'
import Button from '@material-ui/core/Button'

const DisplayButtonWithCozyTheme = () => (
  <MuiCozyTheme>
    <Button>Default button with Cozy theme</Button>
  </MuiCozyTheme>
)
```

All Material-UI component style can be overrides we made an example with [Button](https://github.com/cozy/cozy-ui/pull/676/files#diff-ac2365e2a55682f1f5366e1ad29cedc0R3), you can find:
- [Component on Material-UI Demo](https://material-ui.com/demos/buttons/)
- [Source of component](https://github.com/mui-org/material-ui/blob/e7cf64a48dedfd4e383dd1047e98f4feef582aa3/packages/material-ui/src/Button/Button.js)
- [Name to overrides style of this components is MuiButton](https://github.com/mui-org/material-ui/blob/e7cf64a48dedfd4e383dd1047e98f4feef582aa3/packages/material-ui/src/Button/Button.js#L386)

You can see a demo [here](https://kosssi.github.io/cozy-ui/react/#!/Material-UI)

### CODE REVIEW

**Comment by GoOz** [comment]
Because it brings nothing yet besides unusable components. It's just dead code for now.

**Comment by ptbrowne** [comment]
OK, but it is a start and can be used for improvements. I would prefer to merge early instead of waiting and then having merge conflicts to resolve. Have you planned to work on the theme soon ?

**Comment by CPatchane** [comment]
What is the interest of merging this PR? I thought we should start with a new component as example/POC to start using MUI inside cozy-ui. Here this is just buttons wrapped in a cozy-ui theme, I don't think buttons are the priority here.

File: react/MuiCozyTheme/theme.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+import palette from '../../stylus/settings/palette.json'
+
+import MuiButton from './Buttons/theme'
+
+export const theme = createMuiTheme({
+  typography: {
+    title: {
+      fontFamily: ['TrashHand', 'Roboto', 'Helvetica', 'Arial', 'sans-serif'],
```
Comment (y-lohse) [inline react/MuiCozyTheme/theme.js]: Our main font is `Lato` isn't it?

Comment (GoOz) [reply react/MuiCozyTheme/theme.js]: It is.

**Comment by y-lohse** [comment]
Demo doc with mui available here : https://yannick-lohse.fr/cozy-ui/react/

**Pull request merged**